runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -4264,3 +4264,97 @@ TR::CompilationInfoPerThreadRemote::getCachedIProfilerInfo(TR_OpaqueMethodBlock
       }
    return ipEntry;
    }
+
+void
+TR::CompilationInfoPerThreadRemote::cacheResolvedMethod(TR_ResolvedMethodKey key, TR_OpaqueMethodBlock *method, uint32_t vTableSlot, TR_ResolvedMethod *resolvedMethod, TR_ResolvedJ9JITaaSServerMethodInfo &methodInfo)
```

Comment:
I am seeing `resolvedMethod` being used anywhere in this body

Reply:
Right, it was used in global cache implementation from where I took most of the code. Will remove.